apis/database/v1beta1/rdsinstance_types.go

```diff
@@ -824,8 +824,19 @@ type RDSInstanceParameters struct {
 	//    * First character must be a letter
 	//    * Cannot end with a hyphen or contain two consecutive hyphens
 	// +optional
+	// +crossplane:generate:reference:type=github.com/crossplane-contrib/provider-aws/apis/rds/v1alpha1.DBParameterGroup
 	DBParameterGroupName *string `json:"dbParameterGroupName,omitempty"`
 
+	// DBParameterGroupNameRef is a reference to a DBParameterGroup used to set
+	// DBParameterGroupName.
+	// +optional
+	DBParameterGroupNameRef *xpv1.Reference `json:"dbParameterGroupNameRef,omitempty"`
+
+	// DBParameterGroupNameSelector selects a reference to a DBParameterGroup used to
+	// set DBParameterGroupName.
+	// +optional
+	DBCParameterGroupNameSelector *xpv1.Selector `json:"dbParameterGroupNameSelector,omitempty"`
```

Comment:
I realize this PR is closed, but noticed what appears to be a typo (`C`) in the name of the selector.  Looking for clarification if there's some convention I'm missing, or if in fact this would need addressed.

Reply:
Yes - but er have a other issue that we have import cycle issues in Go because RDS and Database Group today ref/selector for dbsubnetgroup so we are unable to implement this PR as of https://github.com/crossplane-contrib/provider-aws/issues/1042